Sources/GRPC/AsyncAwaitSupport/GRPCAsyncWriterSinkDelegate.swift

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import DequeModule
```

Comment:
This file and type needs `#if` and availability guards same as the `NIOAsyncWriter` has them

Reply:
Do we want  `#if compiler(>=5.6)` or `#if compiler(>=5.5.2) && canImport(_Concurrency)` (and what exactly is the difference)?

Reply:
I think just using the `#if compiler(>=5.6)` is fine. Swift 5.5.2 was a bit weird because Concurrency just got introduced.